resources/lib/bossanova808/utilities.py

```
@@ -236,6 +236,94 @@ def get_addon_version(addon_id: str) -> str | None:
+                    "properties":["playcount"],
+            }
+    }
+
+    query = json.dumps(json_dict)
+    json_response = send_kodi_json(f'Get playcount for {library_type} with dbid: {dbid}', query)
+    result = json_response.get('result')
+    if result:
+        try:
+            play_count = result[result_key]['playcount']
+        except (KeyError, TypeError) as e:
+            Logger.error(f"Could not get playcount")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)